chat-plugins/mafia.js

```diff
@@ -1155,7 +1160,14 @@ exports.commands = {
 				return targetRoom.game.setDeadline('off');
 			} else {
 				target = parseInt(target);
-				if (isNaN(target)) return this.parse(`/help mafia deadline`);
+				if (isNaN(target)) {
+					if (!this.runBroadcast()) return;
+					if (targetRoom.game.dlAt > 0) {
```

Comment:
Why are we changing this? It only shows how much time the timer was originally set for anyways, which Isn't really something we need.

Reply:
No it shows how much time is left until dl

Reply:
Its not setup correctly then.

This will always say X minutes, where X is the original deadline.
If you want to track that, when you would need to track when the timer was started (`Date.now()` when the timer starts), then when calculating, take that, add the amount of time the timer was set for, and subtract the current time (also `Date.now()`). Ex: `(startTime + dlMins * 60000) - Date.now()`